pkg/forklift-api/webhooks/validating-webhook/admitters/provider-admitter.go

```diff
@@ -84,6 +83,19 @@ func (admitter *ProviderAdmitter) validateVDDK() error {
 	return nil
 }
 
+func (admitter *ProviderAdmitter) validateVddkImage() error {
+	if image, found := admitter.provider.Spec.Settings[api.VDDK]; found {
+		if image == "" {
```

Comment:
is that because the UI? does the v2v-cold must have VDDK image? or it will provided with `nil` from UI and expected the same in CLI?